docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,703 @@
+.. `region`: A list of available regions can be obtained using
+`data.rhoas_cloud_providers_regions`
+
++
+.Example `rhoas_kafka` resource
+[source,shell]
+----
+$ terraform {
+  required_providers {
+    rhoas = {
+      source  = "registry.terraform.io/redhat-developer/rhoas"
+      version = "0.3.0"
+    }
+  }
+}
+
+provider "rhoas" {}
+
+resource "rhoas_kafka" "my-instance" {
+  name = "my-instance"
+}
+
+output "bootstrap_server_my-instance" {
+  value = rhoas_kafka.my-instance.bootstrap_server_host
+}
+----
+. Open a terminal and initilize Terraform by entering the following command.
++
+[source,shell]
+----
+$ terraform init
+----
+. Apply the changes by entering the following command.
++
+[source,shell]
+----
+$ terraform apply
+----
+. Terraform displays a message that `rhoas_kafka.my-instance` will be created. Enter *yes*.
+. Verify that the instance has been created in the the table on the *Kafka Instances* page of the {product-kafka} {service-url-kafka}[web console^].
+
+[id="proc-creating-service-account-terraform_{context}"]
+[discrete]
+=== Creating a service account
+
+[role="_abstract"]
+After creating your Kafka instance, you create a service account. To connect your applications or services to a Kafka instance in {product-kafka}, you must first create a service account with credentials.
+
+Use the `rhoas_service_account` resource to manage a service account. Terraform populates the credentials when the resource is applied. This example uses the ‘srvaccnt’ identifier and creates a service account called my-service-account.
+
+.Prerequisites
+
+* You have an offline token that is accessible to Terraform.
+* You have a running Kafka instance in {product-kafka}.
+
+.Procedure
+
+. In your IDE, enter the example `rhoas_service_account` resource to create a service account.
++
+
+.Example `rhoas_service_account` resource
+[source,shell]
+----
+$ resource "rhoas_service_account" "my-instance" {
+  name        = "my-service-account"
+  description = "<description of service account>"
+}
+----
+. Apply the changes by entering the following command. Note that you do not need to run `terraform init` if following on from the Create a Kafka instance step.
++
+[source,shell]
+----
+$ terraform apply
+----
+Terraform populates the generated service account credentials in the `client_id` and `client_secret` fields.
+. Terraform displays a message that `rhoas_kafka.my-service-account` will be created. Enter *yes*.
+. Verify that the service account has been created in the the table on the *Service Accounts* page of the {product-kafka} {service-url-kafka}[web console^].
+
+[id="proc-creating-kafka-topic-terraform_{context}"]
+[discrete]
+=== Creating a Kafka topic
+
+[role="_abstract"]
+Use the `rhoas_topic` resource to create a Kafka topic with default values. This example uses the `topic` identifier and creates the `my-topic` Kafka topic. You set the Kafka ID directly in the resource.
```

Comment:
Similar to the comment in https://github.com/redhat-developer/app-services-guides/pull/601/files#r1029521676, it's initially a bit unclear whether you're showing users a complete, usable example or walking them through a procedure to create one. For example, I obviously need to modify the example resource to add my Kafka instance ID. In that case, it should be an explicit step in the procedure, and not a statement in the abstract.